crates/sui-sdk/tests/tests.rs

```diff
@@ -31,9 +31,8 @@ fn mnemonic_test() {
 /// This test confirms rust's implementation of mnemonic is the same with the Sui Wallet
 #[test]
 fn sui_wallet_address_mnemonic_test() -> Result<(), anyhow::Error> {
```

Comment:
in a follow up pr, this test can be merged with keytool_test (this test class can be deprecated)